Tools/PerformanceTests/automated_test_6_output_2ppc

```diff
@@ -62,3 +62,7 @@ diagnostics.diags_names = diag1
 diag1.intervals = 1
 diag1.file_prefix = "./diags/plt"
 diag1.diag_type = Full
+
+# These options are mainly intended for debug purposes
+warpx.abort_on_warning_threshold = "low"
+warpx.always_warn_immediately = 1
```

Comment:
Is this adding the same two lines to each test?

I would rather like to add them like to add them _in one line_ in `Regression/WarpX-tests.ini` instead:
https://github.com/ECP-WarpX/WarpX/blob/development/Regression/WarpX-tests.ini#L67
```
runtime_params = ...
```

or via
https://github.com/ECP-WarpX/WarpX/blob/53f590c804ea39f040bcbecb5ad7bb6ab939e06e/Regression/prepare_file_ci.py#L72-L76

The rational behind this: it is unnecessarily verbose and we want to add this permanently (like `-Werror`), so that new tests cannot forget to add this line.

Reply:
I agree with you... I'll do that now

Reply:
let's see if all the tests pass...

Reply:
It should be OK now!